src/accounts/accounts.controller.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  ParseIntPipe,
+  BadRequestException,
+  InternalServerErrorException,
+  NotFoundException,
+} from '@nestjs/common';
+import { AccountsService } from './accounts.service';
+import { CreateAccountDto } from './dto/create-account.dto';
+
+@Controller('accounts')
+export class AccountsController {
+  constructor(private readonly accountsService: AccountsService) {}
+
+  @Post()
+  async create(@Body() createAccountDto: CreateAccountDto) {
+    try {
+      return await this.accountsService.create(createAccountDto);
```

Comment:
nit: This function will return all properties of the created account entity in the API response. We should not only expose those properties that will be required. We should especially avoid exposing auto-increment primary keys if possible

Reply:
[Fixed](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/src/settings/settings.controller.ts#L27)